validator/evaluation/docker_evaluation.py

```diff
@@ -21,7 +23,7 @@
 
 
 async def get_evaluation_results(container):
```

Comment:
Here, have we not overwritten the text stuff? So we can only have one working?

Reply:
Done, using same for both. It was just a lazy fix to quickly test without rebuilding the image last night.